src/scripts/react/components/DeclarationDynamicsChart.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from "react";
+import { subDays, format } from "date-fns";
+import { ua } from "date-fns/esm/locale";
+import { Plurals } from "smart-plurals";
+import {
+  ResponsiveContainer,
+  AreaChart,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Tooltip,
+  Area
+} from "recharts";
+
+const { API_ENDPOINT } = window.__CONFIG__;
+const DECLARATIONS_DICT = ["декларація", "декларації", "декларацій"];
+
+const pluralize = Plurals.getRule("uk");
+
+export default class DeclarationsDynamicsChart extends Component {
+  state = {
+    data: []
+  };
+
+  async componentDidMount() {
+    const params = getHistogramParams(
+      document.body.clientWidth > 768 ? 30 : 15
```

Comment:
30? 15? Whaaaat is this?

Reply:
Its days count to display histogram for (as seen from parameter name), we displaying histogram for last 15 days for viewport less than 768px and for 30 days otherwise

Reply:
You can write somth like this for more readable code.
```javascript
const daysCount = document.body.clientWidth > 768 ? 30 : 15;
```
What do you think about this?